extensions/sql-database-projects/src/controllers/mainController.ts

```diff
@@ -56,35 +58,56 @@ export default class MainController implements vscode.Disposable {
+		this.context.subscriptions.push(vscode.commands.registerCommand('sqlDatabaseProjects.newItem', async (node: WorkspaceTreeItem) => { return this.projectsController.addItemPromptFromNode(node); }));
+		this.context.subscriptions.push(vscode.commands.registerCommand('sqlDatabaseProjects.addExistingItem', async (node: WorkspaceTreeItem) => { return this.projectsController.addExistingItemPrompt(node); }));
+		this.context.subscriptions.push(vscode.commands.registerCommand('sqlDatabaseProjects.newFolder', async (node: WorkspaceTreeItem) => { return this.projectsController.addFolderPrompt(node); }));
+
+		this.context.subscriptions.push(vscode.commands.registerCommand('sqlDatabaseProjects.addDatabaseReference', async (node: WorkspaceTreeItem) => { return this.projectsController.addDatabaseReference(node); }));
+		this.context.subscriptions.push(vscode.commands.registerCommand('sqlDatabaseProjects.openContainingFolder', async (node: WorkspaceTreeItem) => { return this.projectsController.openContainingFolder(node); }));
+		this.context.subscriptions.push(vscode.commands.registerCommand('sqlDatabaseProjects.editProjectFile', async (node: WorkspaceTreeItem) => { return this.projectsController.editProjectFile(node); }));
+		this.context.subscriptions.push(vscode.commands.registerCommand('sqlDatabaseProjects.convertToSdkStyleProject', async (node: WorkspaceTreeItem) => { return this.projectsController.convertToSdkStyleProject(node); }));
+		this.context.subscriptions.push(vscode.commands.registerCommand('sqlDatabaseProjects.delete', async (node: WorkspaceTreeItem) => { return this.projectsController.delete(node); }));
+		this.context.subscriptions.push(vscode.commands.registerCommand('sqlDatabaseProjects.exclude', async (node: WorkspaceTreeItem) => { return this.projectsController.exclude(node); }));
+		this.context.subscriptions.push(vscode.commands.registerCommand('sqlDatabaseProjects.changeTargetPlatform', async (node: WorkspaceTreeItem) => { return this.projectsController.changeTargetPlatform(node); }));
+		this.context.subscriptions.push(vscode.commands.registerCommand('sqlDatabaseProjects.validateExternalStreamingJob', async (node: WorkspaceTreeItem) => { return this.projectsController.validateExternalStreamingJob(node); }));
+		this.context.subscriptions.push(vscode.commands.registerCommand('sqlDatabaseProjects.openInDesigner', async (node: WorkspaceTreeItem) => {
+			if (node?.element instanceof TableFileNode) {
+				const tableFileNode = node.element as TableFileNode;
+				const projectNode = tableFileNode.root as ProjectRootTreeItem;
+				const filePath = tableFileNode.fileSystemUri.fsPath;
+				const projectPath = projectNode.project.projectFilePath;
+				const targetVersion = projectNode.project.getProjectTargetVersion();
+				await azdataType.designers.openTableDesigner('MSSQL', {
+					title: tableFileNode.friendlyName,
```

Comment:
in the gif, the editor title changes from "Table1.sql" to "dbo.Table3". Is this expected? It would be better if it doesn't change

Reply:
the Table1.sql is the temporary title until we know which table is being designed. keep using Table1.sql will confuse user if there are multiple tables in it.

Reply:
It would be nice to not have the title change, but if it isn't possible to know the table before opening the editor, then this makes sense

Reply:
I created a task for Aug release. https://github.com/microsoft/azuredatastudio/issues/19600